auth/ante.go

```diff
@@ -74,6 +89,12 @@ func NewAnteHandler(utxoMapper types.UTXOMapper, txIndex *uint16, feeAmount *uin
 			if !res.IsOK() {
 				return ctx, res, true
 			}
+
+			inputs += indenom2
+		}
+
+		if inputs != spendMsg.Denom1+spendMsg.Denom2+spendMsg.Fee {
```

Comment:
need spaces between operands for readability

Reply:
go fmt removed my original spaces for some reason. Will add parenthesis to prevent this for future modifications of ante combined with using go fmt